scripts/VirtualizationTools.ps1

```diff
@@ -0,0 +1,29 @@
+
+choco install -y Microsoft-Windows-Subsystem-Linux -source windowsfeatures

```

Comment:
this does seem duplicative... we should remove the "Enable-" lines... will confirm and update the PR